wetmap/src/components/newModals/setting/view.tsx

```diff
@@ -15,16 +19,41 @@ export default function SettingsView(props: SettingsProps) {
         <ButtonIcon
           icon={<Icon name="chevron-left" />}
           className="btn-lg text-gray ml-4 mt-4"
-          onClick={(e) => {
-            if (props.onClose) {
-              props.onClose(e);
-            }
-          }}
+          onClick={props.onClose}
         />
       </div>
       <div className="mx-10 text-left">
         <h1 className="mt-4 text-bold">{screenData.SettingsPage.header}</h1>
+        <h2 className="ml-4 mt-2 mb-1">{screenData.SettingsPage.subHeading1}</h2>
+        <div className={styles.outline}>
+          <h4 className="ml-8 mb-0 mt-1 text-bold text-dark">{props.profileType}</h4>
+          { props.profileType === 'Diver Account'
+            ? (
+                <span onClick={props.handlePartnerButton}>
+                  <p className="ml-10 mb-1 p-0 text-bold text-primary">{screenData.SettingsPage.notPartnerAccount}</p>
+                </span>
+              )
+            : (
+                <span onClick={props.handlePartnerButton}>
```

Comment:
This can be changed to null,  at this time there are only the diver and partner accounts. Also once a user upgrades to Partner that's it no further account upgrade options are planned to be offered